.github/workflows/sonarqube-scan-pr.yml

```
@@ -0,0 +1,25 @@
+on:
+  # Warning: using the pull_request_target event without the cautionary measures may allow
+  # unauthorized GitHub users to open a “pwn request” and exfiltrate secrets.
+  pull_request_target:
+    types: [opened, synchronize, reopened, labeled]
```

Any reason we should have `labeled` here? Seems like the other triggers are appropriate, but I don't think labeling would affect the scan results
```suggestion
    types: [opened, synchronize, reopened]
```

---

Agreed, I'm ok with removing this.